styles/main.css

```diff
@@ -0,0 +1,526 @@
+
+    color: #3d3d3d;
+}
+
+
+.block-2
+{
+    display: flex;
+
+    width: 100%;
+
+    justify-content: space-between;
+    flex-wrap: wrap;
+}
+
+.item
+{
+    margin-top: 70px;
+
+    -webkit-box-shadow: 3px 3px 5px 0 rgba(120,120,120,1);
+       -moz-box-shadow: 3px 3px 5px 0 rgba(120,120,120,1);
+            box-shadow: 3px 3px 5px 0 rgba(120,120,120,1);
+
+    flex-basis: 22%;
+}
+
+.mini-container
+{
+    padding: 10px 10px;
+}
+
+.item img
+{
+    width: 100%;
+}
+
+.item .button
+{
+    width: 100%;
+    margin: 20px auto;
+
+    cursor: pointer;
+}
+
+h3
+{
+    margin: 0 0 10px;
+    padding: 0;
+
+    color: #093c60;
+}
+
+.bold
+{
+    font-weight: bold;
+}
+
+.text
+{
+    line-height: 1.7;
+}
+
+.more-dogs
+{
+    font-weight: 900;
+    font-size: 1em;
+
+    display: block;
+
+    width: 15%;
+    margin: 80px auto;
+    padding: 12px 30px;
+
+    text-align: center;
+    text-decoration: none;
+
+    color: #093c60;
+    border-radius: 5px;
+    background: #ececec;
+}
+
+/* end of section */
+
+footer
+{
+    padding: 50px 0;
+
+    text-align: center;
+
+    color: #fff;
+    background: #093c60;
+}
+
+footer img,
+footer p
+{
+    width: 25%;
+}
+
+footer p
+{
+    line-height: 2;
+
+    margin: 15px auto;
+
+    color: #fff;
+}
+
+.second
+{
+    margin: 40px 0;
+}
+
+.second a
+{
+    font-weight: bold;
+
+    text-decoration: none;
+
+    color: white;
+}
+
+.second .after:after
+{
+    font-weight: bold;
+
+    margin: 0 20px;
+
+    content: '/';
+}
+
+footer .left:after
+{
+    margin: 0 20px;
+
+    content: '|';
+}
+
+/* Dogs second */
+h2
+{
+    font-size: 2.5em;
+
+    letter-spacing: 1px;
+}
+
+.dogs-second .item
+{
+    margin: 0 0 40px;
+}
+
+
+
+/* ***************checkout****************** */
+form
+{
+    width: 40%;
+}
+
+input
+{
+    font-size: 15px;
+
+    width: 100%;
+    height: 35px;
+    margin-bottom: 30px;
+    padding: 0 10px;
+
+    border: 1px solid #c0c0c0;
+}
+
+label,
+.label
+{
+    font-size: 1.3em;
+    font-weight: bold;
+
+    display: block;
+
+    margin-bottom: 10px;
+
+    color: #093c60;
+}
+
+label[for=location]
+{
+    margin-top: 20px;
+}
+
+input[type=number]::-webkit-inner-spin-button
+{
+    -webkit-appearance: none;
+}
+
+.form-row
+{
+    display: flex;
+
+    justify-content: space-between;
+    flex-wrap: wrap;
+}
+
+.form-row .city-form
+{
+    width: 40%;
+}
+
+.form-row .state-form
+{
+    width: 20%;
+}
+
+.form-row .zip-form
+{
+    width: 30%;
+}
+
+.form-row-2
+{
+    display: flex;
+
+    width: 50%;
+
+    justify-content: flex-start;
+}
+
+/* *******************SELECT SETTINGS******************* */
+
+select
+{
+    font: inherit;
+    line-height: 1.5em;
+
+    display: inline-block;
+
+    -webkit-box-sizing: border-box;
+       -moz-box-sizing: border-box;
+            box-sizing: border-box;
+    width: 8.8em;
+    /* reset */
+    margin: 0;
+    padding: 6px 5px;
+
+    border: 1px solid #c0c0c0;
+    /* styling */
+    background-color: white;
```

Comment:
Since you're using hex values in the css, it's best to stick with that, instead of using the color string. You'd want #FFFFFF here (the case of the letter doesn't matter, as long as you're consistent)